src/scenes/endscreen.ts

```diff
@@ -30,17 +32,40 @@ export class EndScreen extends Phaser.Scene {
         this.scene.start("TitleScreen");
     }
 
-    private displayMessage(): void {
-        let text: string = "You loose";
+    private displayByRole(): void {
+        const { showerOre, text } = this.displayData();
```

Comment:
i like the refactor but now the displayData doesnt display anything anymore so i would rename it